projects/scream/scream_post/format_output.py

```diff
@@ -72,94 +73,163 @@ def make_new_date(dates, year_offset=2015):
     )
-    ds["qv_tendency_due_to_nudging"] = make_placeholder_data(
-        ds.T_mid, "qv_tendency_due_to_nudging", 1e-7
+    parser.add_argument(
+        "nudging_variables",
+        type=str,
+        help="List of nudging variables deliminated with commas",
     )
-    ds["x_wind_tendency_due_to_nudging"] = make_placeholder_data(
-        ds.T_mid, "x_wind_tendency_due_to_nudging", 1e-3
+    parser.add_argument(
+        "--split-horiz-winds",
+        type=bool,
+        default=False,
+        help="Split horiz_winds to u and v",
     )
-    ds["y_wind_tendency_due_to_nudging"] = make_placeholder_data(
-        ds.T_mid, "y_wind_tendency_due_to_nudging", 1e-3
+    parser.add_argument(
+        "--calc-physics-tend",
+        type=bool,
+        default=False,
+        help="Back out physics only tendencies",
     )
-    ds["tendency_of_T_mid_due_to_scream_physics"] = make_placeholder_data(
-        ds.T_mid, "tendency_of_T_mid_due_to_scream_physics", 1e-5
+    parser.add_argument(
+        "--rename-nudging-tend",
+        type=bool,
+        default=False,
+        help="Rename nudging tendencies",
     )
-    ds["tendency_of_qv_due_to_scream_physics"] = make_placeholder_data(
-        ds.T_mid, "tendency_of_qv_due_to_scream_physics", 1e-8
+    parser.add_argument(
+        "--rename-delp",
+        type=bool,
+        default=False,
+        help="Rename psuedo density to pressure thickness",
     )
-    ds["tendency_of_eastward_wind_due_to_scream_physics"] = make_placeholder_data(
-        ds.T_mid, "tendency_of_eastward_wind_due_to_scream_physics", 1e-5
+
+    parser.add_argument(
+        "--rename-lev-to-z",
+        type=bool,
+        default=False,
+        help="Rename psuedo density to pressure thickness",
+    )
+
+    parser.add_argument(
+        "--convert-to-cftime",
+        type=bool,
+        default=False,
+        help="Convert time type to cftime",
     )
-    ds["tendency_of_northward_wind_due_to_scream_physics"] = make_placeholder_data(
-        ds.T_mid, "tendency_of_northward_wind_due_to_scream_physics", 1e-5
+
+    parser.add_argument(
+        "--rename-water-vapor-path",
+        type=bool,
+        default=False,
+        help="Rename VapWaterPath to water_vapor_path",
     )
-    # make new date here because this round of data has year 0001
-    # and causes issues with diagnostics code
-    new_date = make_new_date(ds.time.values)
-    ds = ds.assign_coords(time=new_date)
-    ds = ds.rename({"lev": "z", "ilev": "z_interface"})
-
-    nudging_variables = [
-        "T_mid_tendency_due_to_nudging",
-        "qv_tendency_due_to_nudging",
-        "x_wind_tendency_due_to_nudging",
-        "y_wind_tendency_due_to_nudging",
+    return parser
+
+
+if __name__ == "__main__":
+    parser = _get_parser()
+    args = parser.parse_args()
+    ds = xr.open_mfdataset(args.input_data)
+    nudging_vars = [str(item) for item in args.nudging_variables.split(",")]
+    if args.split_horiz_winds:
```

Comment:
The chain of operations seems like good opportunity for some kind of transform pipeline with a configuration spec.  I think we've generally found that using configurations makes reproducibility artifacts much easier than having runtime flags.  No need for changes though since this works too.